modules/33-services

```diff
@@ -14,8 +14,8 @@ source "$BASE_DIR/framework.sh"
 
 statuses=()
 for key in "${!services[@]}"; do
-    if [ $(systemctl list-unit-files "$key.service" | wc -l) -gt 3 ]; then
-        status=$(systemctl show -p ActiveState --value $key.service);
+    if [ $(systemctl list-unit-files "${key}*" | wc -l) -gt 3 ]; then
```

Comment:
That's the thing, without the `*` it won't be backwards compatible. `systemctl list-unit-files` needs full unit name (or a wildcard) and doesn't use `.service` as a default. So unless you have a better idea for this check I would merge it like this :smiley:.

Reply:
If that's the case, it's fine but take a look at the man page.

<https://man7.org/linux/man-pages//man1/systemctl.1.html>
> Unit commands listed above take either a single unit name
       (designated as UNIT), or multiple unit specifications (designated
       as PATTERN...). In the first case, the unit name with or without
       a suffix must be given. If the suffix is not specified (unit name
       is "abbreviated"), systemctl will append a suitable suffix,
       ".service" by default, and a type-specific suffix in case of
       commands which operate only on specific unit types. For example,

Reply:
Actually, I think everything is ok here because:
> list-unit-files [PATTERN...]

and
> In the first case, the unit name with or without a suffix must be given.

The first case i.e. `UNIT`, not `PATTERN`. 

Well, the most important thing is that it works so I'm merging this :smiley: